packages/import-utils/src/create-sandbox/templates.ts

```diff
@@ -102,6 +103,10 @@ export function getTemplate(
     return "gridsome";
   }
 
+    if (totalDependencies.indexOf("@stackbit/unibit") > -1) {
```

Comment:
@SaraVieira This can just be

```suggestion
    if (totalDependencies.includes('@stackbit/unibit')) {
```